src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +88,7 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!title || !imgUrl || !imdbId || !imdbUrl}
```

Comment:
Create a separate variable that defines if the button is disabled
```suggestion
            disabled={variable}
```